controllers/administration.js

```diff
@@ -584,11 +584,12 @@ const createSystemHandler = (req, res, next) => {
 	});
 };
 
-const getStorageProviders = () => {
-	return [
-		{ label: 'AWS S3', value: 'awsS3' }
-	];
-};
+const getStorageProviders = () => [
+	{
+		label: process.env.SC_SHORT_TITLE,
```

Comment:
res.locals.theme = {
        title: process.env.SC_TITLE || "HPI Schul-Cloud",
        short_title: process.env.SC_SHORT_TITLE || "Schul-Cloud",
    };

is set in app.js, that should be reused.

Reply:
reverted back to res.locals.theme